src/raftstore/store/fsm/peer.rs

```
@@ -1439,10 +1439,13 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
     fn on_ready_change_peer(&mut self, cp: ChangePeer) {
         let change_type = cp.conf_change.get_change_type();
         if let Err(e) = self.fsm.peer.raft_group.apply_conf_change(&cp.conf_change) {
-            panic!(
-                "{} apply conf change {:?} fails: {:?}",
-                self.fsm.peer.tag, cp, e
-            );
+            match e {
+                raft::Error::Exists(_, _) | raft::Error::NotExists(_, _) => {}
```

As a bugfix, please add some comments and a test to cover the change.

---

Could add some comment here to explain why these errors can be ignored?